### ISSUE TITLE
[Bug] Unable to run axe hints via API w/ Puppeteer - Chrome - AnalyzerError: Missing or incompatible dependencies

### ISSUE THREAD

**Issue by bearnecessities** [webhintio/hint#4362]
# 🐛 Bug report

## Description

I'm currently unable to successfully run axe hints using Puppeteer. 
I get the following error - `AnalyzerError: Missing or incompatible dependencies`.
In the past I've successfully used the puppeteer connector with chrome, in the same manner.

I have a public repo where the issue is reproducible for me - https://github.com/bearnecessities/just-testing
just clone then `npm i && npm run accessibility-tests`

I believe there may be an issue here, but please advise if I'm doing something incorrectly here.

## Environment

* __`hint` version:__ v6.1.3
* __`Node.js` version:__ v12.19.1
* __`OS`:__ macOS Catalina v10.15.7
* __`Chrome`:__  89.0.4389.72 (Official Build) (x86_64)

## `webhint` configuration

<details>
<summary>webhint’s configuration</summary>

```js
const userConfig = {
  connector: {
    name: 'puppeteer',
    options: {
      browser: 'Chrome',
      headless: true
    }
  },
  formatters: ['html', 'summary'],
  hints: {
    'axe/aria': 'error',
    'axe/color': 'error',
    'axe/forms': 'error',
    'axe/keyboard': 'error',
    'axe/language': 'error',
    'axe/name-role-value': 'error',
    'axe/other': 'error',
    'axe/parsing': 'error',
    'axe/semantics': 'error',
    'axe/sensory-and-visual-cues': 'error',
    'axe/structure': 'error',
    'axe/tables': 'error',
    'axe/text-alternatives': 'error',
    'axe/time-and-media': 'error'
  },
  hintsTimeout: 60000
};

```

</details>

## Debug output

<details>
<summary>webhint’s raw output</summary>

```text
     AnalyzerError: Missing or incompatible dependencies
      at validateResources (node_modules/hint/dist/src/lib/analyzer.js:22:15)
      at Function.create (node_modules/hint/dist/src/lib/analyzer.js:67:9)
      at Object.runHints (accessibility/hints.js:32:34)
      at Context.<anonymous> (accessibility/test-spec.js:6:35)
      at processImmediate (internal/timers.js:461:21)



```

</details>

**Comment by hxlnt** [comment]
Thanks for the feedback, @bearnecessities! We're going to take a look at this and see if we can repro. ✌

**Comment by bearnecessities** [comment]
@antross @hxlnt Hey, just wanted to circle back and see if the issue was reproducible on your end. Is there any additional information that I can provide that would be helpful?

**Comment by antross** [comment]
Hey @bearnecessities, sorry it took a bit to get to this.

I was able to repro the issue. Once I did, I was able to debug to find out the specific missing dependency was `axe/other`. This information is on the `missing` property of the thrown exception, but we really should put it in the message too so scenarios like this are easier to debug. Probably just a small edit needed here: https://github.com/webhintio/hint/blob/main/packages/hint/src/lib/types/analyzer-error.ts

Long story short, the current public version of `@hint/hint-axe` doesn't have an `axe/other` sub-hint (though it does in the `main` branch so one should be available in the next release). For now, removing `axe/other` from the configuration passed to the webhint Node API fixed the issue.

**Comment by antross** [comment]
@sarvaje what do you think about this?

> This information is on the `missing` property of the thrown exception, but we really should put it in the message too so scenarios like this are easier to debug. Probably just a small edit needed here: https://github.com/webhintio/hint/blob/main/packages/hint/src/lib/types/analyzer-error.ts

**Comment by antross** [comment]
@bearnecessities I'm guessing just being able to specify something like `axe` or `axe/*` would have been easier for your use case as well, correct?

**Comment by bearnecessities** [comment]
@antross thanks for looking into this! It would definitely be awesome to be able to specify something like `axe/*`

**Comment by khmyznikov** [comment]
@vidorteg 

**Config:**

```
const userConfig: UserConfig = {
      hints: {
        "https-only": "information",
        "manifest-exists": "information",
        "manifest-is-valid": "information",
        "manifest-file-extension": "information",
        "meta-theme-color": "information",
        "meta-viewport": "information",
        "manifest-app-name": "information",
        "apple-touch-icons": "information"
      },
      connector: {
        name: "puppeteer",
        options: {
            browser: "Edge",
            headless: true,
        }
      },
      hintsTimeout: 30 * 1000,
      formatters: [
        'json'
      ]
  };
```
**Error:**
```{"name":"AnalyzerError","status":"ResourceError","resources":{"formatters":[null],"hints":[null,null,null,null,null,null],"incompatible":[],"missing":["hint-https-only","hint-meta-theme-color"],"parsers":[]}}```

**Version:**
`"hint": "7.1.3"`
